android9/overlay/etc/init/mount-android.conf

```diff
@@ -11,10 +11,8 @@ script
     mkdir -p /dev/cpuset
     mount none /dev/cpuset -t cpuset -o nodev,noexec,nosuid
 
-    mount -o bind /android/system/system /system
-
     /var/lib/lxc/android/mount-android.sh
 
-    mount -o bind /android/data /data
-    mount -o bind /android/system /var/lib/lxc/android/rootfs
+    mount -o bind /mnt /var/lib/lxc/android/rootfs/mnt
+    mount -o bind /metadata /var/lib/lxc/android/rootfs/metadata
```

Comment:
We can't be sure that /metadata mount point always exist in the built system image (it doesn't exist for me). Probably it is better to add if exist condition here.

Reply:
That I can take care of.